### PR TITLE
Add documentation to the `cocoapods` action

### PR DESCRIPTION
**Pull request by davidlormor**
Documentation for the available options/params that can be passed to the `cocoapods` fastlane action

### CODE REVIEW

File: fastlane/docs/Actions.md [hunk 1]
```diff
@@ -41,6 +41,21 @@ If you use [CocoaPods](http://cocoapods.org) you can use the `cocoapods` integra
...
 ```
 
+More options are available:
+
+```ruby
+cocoapods(
+  clean: true,                                    # Remove SCM directories
+  integrate: true,                                # Integrate the Pods libraries into the Xcode project(s)
+  repo_update: false,                             # Run `pod repo update` before install
+  silent: false,                                  # Show nothing
```
Comment (KrauseFx) [inline fastlane/docs/Actions.md]: `Show nothing` might not be clear, let's rephrase it

**Comment by KrauseFx** [comment]
Looks great, thanks 👍  I just added one comment

**Comment by davidlormor** [comment]
Awesome - I took the comments directly from the documentation in the code...I'll see if I can make things a little more descriptive and push the update. BTW, kudos on all this tooling, @KrauseFx - I don't know that I could ship my apps without it!

File: fastlane/docs/Actions.md (hunk 1 above)
Comment (davidlormor) [inline fastlane/docs/Actions.md]: How about `Execute command without logging output`, @KrauseFx?

Comment (KrauseFx) [inline fastlane/docs/Actions.md]: Yeah perfect 👍

**Comment by KrauseFx** [comment]
Ping me when you updated the docs, so I can merge 👍

**Comment by davidlormor** [comment]
@KrauseFx added the requested text change, and updated the `description` field for the `available_options` docs to match.

**Comment by KrauseFx** [comment]
This is great, thanks 👍

**Pull request merged**